src/CLI/clitree/cli-xml/sflow.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+Copyright 2019 Dell, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<CLISH_MODULE
+    xmlns="http://www.dellemc.com/sonic/XMLSchema"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:xi="http://www.w3.org/2001/XInclude"
+    xsi:schemaLocation="http://www.dellemc.com/sonic/XMLSchema
+    http://www.dellemc.com/sonic/XMLSchema/clish.xsd"
+    >
+
+<VIEW name="enable-view">
+   <COMMAND name="show sflow" help="Show sFlow information">
+         <ACTION> python $SONIC_CLI_ROOT/sonic-cli-sflow.py get_sflow show_sflow.j2 </ACTION>
+   </COMMAND>
+   <COMMAND name="show sflow interface" help="Show sFlow interface information">
+         <ACTION> python $SONIC_CLI_ROOT/sonic-cli-sflow.py get_sflow_interface show_sflow_intf.j2 </ACTION>
+   </COMMAND>
+</VIEW>
+  <VIEW name="configure-view">
+    <COMMAND name="sflow" help="enable sflow">
+      <PARAM name="collector" help="Collector name" ptype="SUBCOMMAND" mode="subcommand" optional="true">
+      	<PARAM name="collector-name" help="Collector name" ptype="STRING"/>
+      	<PARAM name="collector-ip" help="Collector IP address" ptype="STRING"/>
+      	<PARAM name="collector-port" help="Collector port number" ptype="UINT" optional="true"/>
+      </PARAM>
+      <PARAM name="agent-id" help="Collector agent interface" ptype="SUBCOMMAND" mode="subcommand" optional="true">
+      	<PARAM name="interface" help="Interface name" ptype="STRING"/>
+      </PARAM>
+      <ACTION builtin="clish_nop"/>
+      <PARAM name="polling-interval" help="Collector polling interval" ptype="SUBCOMMAND" mode="subcommand" optional="true">
+      	<PARAM name="interval" help="Interval" ptype="UINT"/>
+      </PARAM>
+    </COMMAND>
+    <COMMAND name="no sflow" help="disable sflow">
+      <PARAM name="collector" help="Collector name" ptype="SUBCOMMAND" mode="subcommand" optional="true">
+      	<PARAM name="collector-name" help="Collector name" ptype="STRING"/>
+      </PARAM>
```

Comment:
Are you missing `no sflow agent-id` and `no sflow polling-interval`?

Reply:
will add these.